sdk/ai/azure-ai-inference/tests/sample1.prompty

```diff
@@ -0,0 +1,30 @@
+---
+name: Basic Prompt
+description: A basic prompt that uses the GPT-3 chat API to answer questions
+authors:
+  - author_1
+  - author_2
+model:
+  api: chat
+  configuration:
+    azure_deployment: gpt-4o-mini
+  parameters:
+    temperature: 1
+    frequency_penalty: 0.5
+    presence_penalty: 0.5
```

Comment:
You show the parameters here, but they are not really applied by the Inference SDK... so this is a bit misleading. I would remove everything that is not relevant to usage of Prompty with Inference SDK

Reply:
These 3 parameters demonstrates how Prompty passes the parameters from Prompty files into Inference SDK -- it's from Bozhong's doc which suggests the following pattern, `prompty.config` is the parameters. Let's discuss together with the PMs

        response = chat_completions_client.complete(
          messages = messages,
          model = prompty.model_name,
          **prompty.config
        )

Reply:
PM feedback: it's fine to keep these parameters as Bozhong suggested